src/main/webapp/clusters-view-physical/clusters-view.html

```diff
@@ -28,7 +28,16 @@
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <h2 class="h5">Latest status Events:</h2>
       </div>
-      <ul class="list-group">
+      <ul class="list-group"  ng-if="gridEvents.length > 0" >
+        <li class="list-group-item" ng-repeat="event in gridEvents | orderBy:'when'">
+          <h3 class="list-group-item-heading">
+            <span ng-if="event.level=='WARN'"  class="icon-class pficon-warning-triangle-o"></span>
```

Comment:
we can later dress up these events a bit and use a function here i.e isWarn(), isInfo() etc